src/altinn-app-frontend/src/features/form/components/FullWidthWrapper.tsx

```diff
@@ -19,14 +20,25 @@ const useStyles = makeStyles({
       marginRight: '-96px',
     },
   },
+  consumeBottomPadding: {
+    marginBottom: '-24px',
+    '@media (min-width: 768px)': {
+      marginBottom: '-36px',
+    },
+  },
 });
 
-export function FullWidthWrapper({ children }: IFullWidthWrapperProps) {
+export function FullWidthWrapper({
+  children,
+  onBottom,
+}: IFullWidthWrapperProps) {
   const classes = useStyles();
 
   return (
     <div
-      className={classes.fullWidthWrapper}
+      className={`${classes.fullWidthWrapper} ${
+        onBottom ? classes.consumeBottomPadding : ''
+      }`}
```

Comment:
Would suggest to use `classnames` helper for this conditional class

```jsx
import cn from 'classnames';
```

```suggestion
      className={cn(classes.fullWidthWrapper, {
        [classes.consumeBottomPadding]: isOnBottom
      }}
```